src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingController.kt

```diff
@@ -108,15 +108,16 @@ class VurderingController(
     @PostMapping("gjenbruk-enkelt-vilkår")
     fun gjenbrukEnkeltVilkår(
         @RequestBody request: EnkeltVilkårForGjenbrukRequest,
-    ): Ressurs<VilkårsvurderingDto> {
+    ): Ressurs<VilkårsvurderingDto?> {
         val behandlingForGjenbruk = gjenbrukVilkårService.finnBehandlingerForGjenbruk(request.behandlingId).first()
 
         tilgangService.validerTilgangTilBehandling(behandlingForGjenbruk.id, AuditLoggerEvent.ACCESS)
         tilgangService.validerTilgangTilBehandling(request.behandlingId, AuditLoggerEvent.UPDATE)
         tilgangService.validerHarSaksbehandlerrolle()
 
         val vilkårsVurderingForGjenbruk = gjenbrukVilkårService.gjenbrukInngangsvilkårVurdering(request.behandlingId, behandlingForGjenbruk.id, request.vilkårId)
-        val vilkårsvurderingDto = vilkårsVurderingForGjenbruk.tilDto()
-        return Ressurs.success(vilkårsvurderingDto)
+        vilkårsVurderingForGjenbruk?.let {
+            return Ressurs.success(it.tilDto())
+        } ?: return Ressurs.failure("Vilkårsvurdering for gjenbruk finnes ikke")
```

Comment:
Jeg tenker at det er bedre at servicen kaster `brukerfeil` direkte enn at vi sjekker for null i controlleren, eller?

Reply:
er det en brukerfeil ?

Reply:
Rent teknisk er det riktig - alle den tid brukerFeil ikke logges som error. 
Så bør vi heller ha en v3 av dette hvor vi unngår å "tilby" gjenbruk der det ikke gir mening..?

Reply:
ja, blir litt rart å kalle det brukerfeil. Hadde vært fint å unngå gjenbruk hvis det ikke finnes.

Reply:
unngår å kaste brukerfeil eller noe sånt, og unngår å gjenbruke når man ikke kan